rdkit/Chem/AtomPairs/Pairs.py

```diff
@@ -16,14 +16,13 @@
 Definition and Applications" JCICS 25, 64-73 (1985).
 
 """
-from rdkit.DataStructs import IntSparseIntVect
-from rdkit import Chem
+from rdkit import DataStructs
 from rdkit.Chem import rdMolDescriptors
 from rdkit.Chem.AtomPairs import Utils
-from rdkit import DataStructs
 
-from rdkit.Chem.rdMolDescriptors import GetAtomPairFingerprint, GetHashedAtomPairFingerprint
+GetAtomPairFingerprint = rdMolDescriptors.GetAtomPairFingerprint
```

Comment:
Though I have no particular objection to this change, I wonder why you made it.
Is the `from x import y` pattern considered bad now?